app/models/publisher.rb

```diff
@@ -303,6 +304,10 @@ def in_new_ui_whitelist?
     partner?
   end
 
+  def accepted_policy_agreements?
```

Comment:
We have a lot of methods in the publisher model that aren't necessarily directly related.
I feel like keeping this in the policy_agreement relationship will lead to more separation of concerns. 

```ruby
# What people will call
publisher.policy_agreement.accepted?
```

```ruby
# app/models/policy_agreement.rb
def accepted?
  accepted_publisher_tos && accepted_publisher_privacy_policy
end
```